.github/workflows/user-study-beta-deployment.yml

```diff
@@ -0,0 +1,35 @@
+#
+# This source file is part of the Stanford LLM on FHIR project
+#
+# SPDX-FileCopyrightText: 2025 Stanford University
+#
+# SPDX-License-Identifier: MIT
+#
+
+
+name: User Study Beta Deployment
+on:
+  push:
+    branches:
+      - user-study-deployment
```

Comment:
Note: This workflow should only run manually. Currently, it triggers on every push to allow testing of TestFlight releases.